alarm/kodi-rbp-git/PKGBUILD

```diff
@@ -44,10 +44,10 @@ install:' -i tools/EventClients/Makefile.in
 }
 
 build() {
-  cd "${srcdir}/xbmc-${_commit}"
+  cd "${srcdir}/xbmc-${_tag}"
 
   # Bootstrapping
-  MAKEFLAGS=-j1 ./bootstrap
+  MAKEFLAGS=-j5 ./bootstrap
```

Comment:
Do not do this. Hard-coding that bootstrap to more cores than other users might have is bad practice. -j1 was set because problems with mutli-process in the past. At worst, this should be removed.